app/services/signbank_export.rb

```diff
@@ -0,0 +1,68 @@
+class SignbankExport
+  include Rails.application.routes.url_helpers
+  class URLForbiddenCharacterError < StandardError; end
+
+  SEPARATOR = "|".freeze
+  QUERY = <<~SQL.squish.freeze
+    SELECT
+      array_to_string(array_agg(DISTINCT videos.id), '|') AS videos,
```

Comment:
```suggestion
      array_to_string(array_agg(DISTINCT videos.id), '#{SEPARATOR}') AS videos,
```

You could also use a string format operation here, but I think if you've got the separator in a const you should use it in the query string as well.